pwnlib/elf/elf.py

```diff
@@ -464,9 +464,6 @@ def _describe(self, *a, **kw):
             self.checksec(*a, **kw)
         )
 
-    def __repr__(self):
```

Comment:
Can we have this back?  Thanks!

Reply:
It is already there (the function was defined twice 🤣)